lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -730,6 +801,16 @@ states:
             targetJourney: INELIGIBLE
             targetState: INELIGIBLE
 
+  CRI_DRIVING_LICENCE_AUTH_SOURCE_CHECK_AFTER_PYI_ESCAPE:
+    response:
+      type: cri
+      criId: drivingLicence
+    parent: CRI_STATE
+    events:
+      next:
+        targetJourney: EVALUATE_SCORES
+        targetState: START
+
```

Comment:
This is for same session enhanced verification mitigation. If the user backs out of the drivingLicence CRI  we'll get an access-denied event. At written this will fail the users journey. But we could instead make changes to the reset-session-identity lambda to clear out the DCMAW VC and offer the user F2F as another way to achieve a verification score.
We could potentially do this for any non CI attracting failure from driving licence.